.github/workflows/check-move-backend-pull-request.yml

```diff
@@ -31,3 +31,6 @@ jobs:
 
       - name: Run move-vm-backend tests
         run: cargo test -p move-vm-backend --features build-move-projects-for-test
+
+      - name: Run move-vm-backend-common tests
```

Comment:
We have another workflow for the `language/` part of the repo where the core part of MoveVM is located.

Since these `move-vm-backend-xxx` projects are only using the `language/` part - we don't really need to trigger the whole workspace test suite - since it doesn't affect it.

If it makes sense...